app/helpers/previews_helper.rb

```diff
@@ -6,4 +6,8 @@ def download_text(dataset, datafile)
       t(".download_file")
     end
   end
+
+  def is_numeric(string)
+    true if Float(string) rescue false
```

Comment:
Nice solution. It's still going to fail on things like "10k" or "£50,000" but it's probably better than blind assumption.